### DISCUSSION TITLE
v1.0.0 - Signed Release

### DISCUSSION THREAD

**Discussion by EricGoldsteinNz** [EricGoldsteinNz/tadpole#54] [Announcements]
Small bug and wording fixes
Code now officially signed

<hr /><em>This discussion was created from the release <a href='https://github.com/EricGoldsteinNz/tadpole/releases/tag/v1.0.0'>v1.0.0 - Signed Release</a>.</em>

**Comment by bn40400** [comment]
All I have to say is that you nailed it on this release! Works great and Bravo! Thank you very much (Seriously, your work is much appreciated)!  Great job on the functionality of Tadpole!

**Reply by rafitav** [reply]
hola buen dia , por favor sera que lo puedes subir en 32 bits
